VOKUtilities.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name             = "VOKUtilities"
-  s.version          = "0.12.0"
+  s.version          = "0.12.1"
```

Comment:
I'd recommend making this `0.13` if you're doing the main compilation in Swift 4.2 mode

Reply:
Yep good call